Configuration/TypoScript/ContentElement/Image.typoscript

```diff
@@ -11,6 +11,9 @@ tt_content.image {
                         10 {
                             references.fieldName = image
                             as = files
+                            processingConfiguration {
```

Comment:
I think we should not add commented out code

Reply:
@twoldanski, I generally agree with you but I see an exception here since this is not "programming" code but rather configuration. And I wanna show the possibilities the integrator has here. (EXT:powermail does the same btw.)

Reply:
Yeah, I suspected as much ;) but I think still be a little bit confusing in Typoscript, I recommend to add info to the processor like here https://github.com/TYPO3-Headless/headless/blob/master/Classes/DataProcessing/RootSitesProcessor.php
& ideally to the documentation part of extension

Reply:
Alright, I've just moved the example to the processor class

Proper documentation I will add to the 3.x version of this change, #487